common/changes/@itwin/one-click-lca-react/oclca-rbac-error-message_2022-05-26-22-32.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/one-click-lca-react",
+      "comment": "Update error message notification on unauthorized user triggering carbon calculation jobs. ",
```

Comment:
why are there 3 changelog entries here?

Reply:
Previous change logs were not merged somehow.

Reply:
previous as in from other prs?
you should delete the ones you dont need